### ISSUE TITLE
build calla lib

### ISSUE THREAD

**Issue by manang** [capnmidnight/Calla#106]
Hi,
I'm trying to compile the calla lib on my pc (mac os x)

my steps:

`git clone https://github.com/capnmidnight/Calla.git
cd Calla
git submodule init 
git submodule update
cd Calla
npm install
npm run build`

Inside package.package I corrected:
`        "build": "copy README.md .. && rollup -c --environment BUILD:production",`
to
`        "build": "cp README.md .. && rollup -c --environment BUILD:production",`

but I obtain the following error:

`npm run build                                                     

> calla@0.11.0 build /Users/manangel/collab/calla/node_modules/calla_lib/Calla
> cp README.md .. && rollup -c --environment BUILD:production

src/index.js → dist/calla.js, ../Calla-Site/scripts/lib/calla.js, dist/calla.cjs.js...
[!] Error: Could not resolve '../omnitone/src/omnitone.js' from lib/resonance-audio/src/listener.js
Error: Could not resolve '../omnitone/src/omnitone.js' from lib/resonance-audio/src/listener.js
    at error (/Users/manangel/collab/calla/node_modules/calla_lib/Calla/node_modules/rollup/dist/shared/rollup.js:5211:30)
    at ModuleLoader.handleResolveId (/Users/manangel/collab/calla/node_modules/calla_lib/Calla/node_modules/rollup/dist/shared/rollup.js:18258:24)
    at /Users/manangel/collab/calla/node_modules/calla_lib/Calla/node_modules/rollup/dist/shared/rollup.js:18250:22
    at async Promise.all (index 0)
    at ModuleLoader.fetchStaticDependencies (/Users/manangel/collab/calla/node_modules/calla_lib/Calla/node_modules/rollup/dist/shared/rollup.js:18248:34)
    at async Promise.all (index 0)
    at ModuleLoader.fetchModule (/Users/manangel/collab/calla/node_modules/calla_lib/Calla/node_modules/rollup/dist/shared/rollup.js:18225:9)
    at async Promise.all (index 0)
    at ModuleLoader.fetchStaticDependencies (/Users/manangel/collab/calla/node_modules/calla_lib/Calla/node_modules/rollup/dist/shared/rollup.js:18248:34)
    at async Promise.all (index 0)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! calla@0.11.0 build: `cp README.md .. && rollup -c --environment BUILD:production`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the calla@0.11.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
`

How can I solve this issue?
Thanks
Angelo

**Comment by manang** [comment]
is this include important in lib/resonance-audio/src/listener.js?
import * as Omnitone from "../omnitone/src/omnitone.js";

thanks

Angelo

**Comment by manang** [comment]
ok, I found the problem:
`git clone https://github.com/capnmidnight/Calla.git`
` cd Calla`
`git submodule update --init --recursive`
`cd Calla`
`npm install`
`npm run build`

but I don't undestand why in package.json you need to copy README.md to the root folder.
Thanks

**Comment by manang** [comment]
#107

**Comment by capnmidnight** [comment]
README.md is copied from the Calla directory (which is the root for the NPM package) to the git repository root directory, so that the README only needs to be updated in one place.

Omnitone is a dependency of Resonance Audio. Resonance Audio provides a high-quality audio spatialization system for Calla. Both Omnitone and Resonance Audio are Google projects that were abandoned about 3 years ago. The core of their code was good, but they had significant bugs that I had to correct. Given that the projects are abandoned and they have PRs that are open, I forked the projects and didn't attempt to get my changes merged upstream.

I've merged your PR.
